rio_tiler/io/stac.py

```diff
@@ -337,22 +329,29 @@ def point(
 
     def stats(
         self,
-        assets: Union[Sequence[str], str],
         pmin: float = 2.0,
         pmax: float = 98.0,
+        hist_options: Dict = {},
+        assets: Union[Sequence[str], str] = None,
```

Comment:
This change was made to accommodate the ABC class and avoid changing the `args`.

Note: Breacking change